index.html

```diff
@@ -839,33 +841,33 @@
               <var>acceptPromise</var> with that error and terminate this
               algorithm.
               </li>
-              <li>Determine which <a>payment handlers</a> support
-              <var>identifier</var>. For each resulting payment handler, if
-              payment method specific capabilities supplied by the payment
-              handler match those provided by <var>data</var>, the payment
-              handler matches.
+              <li>Add each <a>payment handler</a> that supports
```

Comment:
I agree with @adrianba's formulation from some time ago:

 1) Build a set of handlers based on PMI matching
 2) Then potentially remove that set based on some other factors

I can think of at least three factors a user agent might taken into account
to remove a payment handler:

 * Payment method specific capability matching
 * Payment method manifest statements
 * Security considerations (e.g., known malicious code or origin)
 
Marcos' statement is broad enough to accommodate these factors:
   "Add each <a>payment handler</a> that supports <var>identifier</var> into <var>handlers</var>."

But I would like to be more explicit about the factors that inform the user agent's decision.
For example:

<pre>
  <li>For each <a>payment handler</a> that supports <var>identifier</var>:
    <ol>
        <li>If the <a>payment handler</a> is not able or authorized to handle
              the payment request, continue. Note: Payment method-specific
             <var>data</var> and information provided through other mechanisms
             inform the user agent's determination.</li>
        <li>Else, add <a>payment handler</a> into <var>handlers</var>.</li>
    </ol>
  </li>
</pre>